--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,15 +5,13 @@
 version = "0.7.0"
 requires-python = ">=3.11"
 dependencies = [
-    "cadquery_ocp_proxy >= 8.0, < 9.0",
+    "ocp3x >= 0.1, < 0.2",
     "svgelements >= 1.9.1, <2",
 ]
 
 
 [project.optional-dependencies]
 dev = [
-    "cadquery_ocp_novtk >= 8.0, < 9.0",
-    "cadquery-ocp-stubs >= 8.0, < 9.0",
     "pytest",
     "coverage",
     "mypy",
